### PR TITLE
Ignore comments

### PR DESCRIPTION
**Pull request by AFriemann**
Regarding issue https://github.com/dennisstritzke/ipsec_exporter/issues/18

Never wrote any go before, so please if you have any suggestions @dennisstritzke I'm happy to oblige

### CODE REVIEW

File: ipsec/configuration_test.go [hunk 1]
```diff
@@ -153,6 +153,20 @@ func TestExtractLines(t *testing.T) {
 	checkInput(t, inputSliced, 3, "Third")
 }
 
+func TestIgnoreComments(t *testing.T) {
+	input := "First\n# Second\n\n#Third\nFourth\n# \nFifth"
+	inputSliced := dummyIpSecConfigLoader().extractLines(input)
+
+	if len(inputSliced) != 4 {
+		t.Errorf("Expected output to have 2 items, but has %d", len(inputSliced))
```
Comment (dennisstritzke) [inline ipsec/configuration_test.go]: I think you meant `4` here.

**Comment by dennisstritzke** [comment]
Thanks for actually submitting and since you are from the Rhein-Main region as I am: Gude, thank you!

Could you resolve the minor comments on your changes? Also could you please add a line to the `CHANGELOG.md`? It should be something like the following:
```
## [Unreleased]
### Changed
- The exporter drops comments found in the `ipsec.conf`
```

File: ipsec/configuration_test.go (hunk 1 above)
Comment (AFriemann) [reply ipsec/configuration_test.go]: indeed

**Comment by AFriemann** [comment]
Ok, just took the changelog entry from your comment and fixed the description in the test

**Pull request merged**